### PR TITLE
Add autofix to at-rule-semicolon-newline-after

### PR DESCRIPTION
**Pull request by modosc**
> Which issue, if any, is this issue related to?

#2829

> Is there anything in the PR that needs further explanation?

i'm not sure whether this is actually the correct/expected behavior - see the conversation in #2829. i just wanted to start getting these fixes in one by one to get the conversation going forward.

### CODE REVIEW

File: lib/rules/at-rule-semicolon-newline-after/__tests__/index.js [hunk 1]
```diff
@@ -79,42 +79,42 @@ testRule(rule, {
...
-    {
-      code: '@charset "UTF-8"; a {};',
+      fixed: '@charset "UTF-8";\n a {};',
       message: messages.expectedAfter(),
       line: 1,
       column: 18
     },
     {
       code: "a{\n@extend .b; @extend .c\n}",
+      fixed: "a{\n@extend .b;\n @extend .c\n}",
```
Comment (hudochenkov) [inline lib/rules/at-rule-semicolon-newline-after/__tests__/index.js]: Can we have the same example, but with `\r\n` newlines, please?

Comment (modosc) [reply lib/rules/at-rule-semicolon-newline-after/__tests__/index.js]: sure - in addition to this one or as a replacement?

**Comment by modosc** [comment]
@jeddy3 i'll fix that with @hudochenkov 's request as well. should i rebase after that since this is so out of date?

File: lib/rules/at-rule-semicolon-newline-after/__tests__/index.js (hunk 1 above)
Comment (hudochenkov) [reply lib/rules/at-rule-semicolon-newline-after/__tests__/index.js]: In addition, please.

**Comment by hudochenkov** [comment]
If you rebase, don't squash the latest changes in this PR, please. So we can see how did you address our comments.

**Comment by modosc** [comment]
ok - not rebasing, just addressed the changes.

**Comment by modosc** [comment]
ugh - i lost the original branch on github - i just pushed it again but this pr isn't picking it up. should i just open a new pr and reference this one?

**Comment by modosc** [comment]
reopened as #3450

**Pull request closed without merging**